.travis/build-mac.bash

```diff
@@ -0,0 +1,23 @@
+brew install ccache glew ninja qt
+export CCACHE_SLOPPINESS=pch_defines,time_macros
+export CMAKE_PREFIX_PATH=/usr/local/opt/qt5/
+export PATH="/usr/local/opt/ccache/libexec:$PATH"
+
+# Setup vulkan and gfx-rs/portability
+curl -sLO https://github.com/gfx-rs/portability/releases/download/latest/gfx-portability-macos-latest.zip
+unzip gfx-portability-macos-latest.zip
```

Comment:
unzip -: gfx-portability-macos-latest.zip
Source: https://unix.stackexchange.com/questions/319622/any-way-to-override-path-traversal-protection-in-unzip/319623#319623